src/base/search/searchpluginmanager.cpp

```diff
@@ -528,7 +528,7 @@ void SearchPluginManager::parseVersionInfo(const QByteArray &info)
         if (list.size() != 2) continue;
 
         const auto pluginName = QString::fromUtf8(list.first().trimmed());
-        const PluginVersion version = PluginVersion::tryParse(list.last().trimmed(), {});
+        const PluginVersion version = PluginVersion::parse(QString::fromLatin1(list.last().trimmed()));
```

Comment:
Suggestions about `auto` and `fromString()` are optional.

Reply:
> And I would rename `parse()` to `fromString()` so that it explicitly indicates the fact that it constructs an instance of `Version`.

I had the same thought and it is done.